RelatedPerson/RelatedPerson-FoetusToRyanneBoulder-Example.json

```diff
@@ -0,0 +1,21 @@
+{
+  "resourceType": "RelatedPerson",
+  "id":"RelatedPerson-FoetusToRyanneBoulder-Example",
+  "patient": {
+    "identifier": {
+      "system": "https://fhir.nhs.uk/Id/nhs-number",
+      "value": "9449307687"
```

Comment:
This should be the identifier for the foetus (RelatedPerson.identifier should be the identifier for the mother)

Reply:
Since Foetus is patient, the mother is now considered as RelatedPerson. Thus, the resource is now for the mother. 
RelatedPerson-RyanneBoulder-Example